tfe/data_source_workspace_ids_test.go

```diff
@@ -427,3 +493,76 @@ data "tfe_workspace_ids" "good" {
   organization = tfe_workspace.foo.organization
 }`, rInt, rInt, rInt, rInt)
 }
+
+func testAccTFEWorkspaceIDsDataSourceConfig_excludeTags(rInt int) string {
+	return fmt.Sprintf(`
+resource "tfe_organization" "foobar" {
+  name  = "tst-terraform-%d"
+  email = "admin@company.com"
+}
+
+resource "tfe_workspace" "foo" {
+  name         = "workspace-foo-%d"
+  organization = tfe_organization.foobar.id
+  tag_names    = ["good", "happy"]
+}
+
+resource "tfe_workspace" "bar" {
+  name         = "workspace-bar-%d"
+  organization = tfe_organization.foobar.id
+  tag_names    = ["good"]
+}
+
+resource "tfe_workspace" "dummy" {
+  name         = "workspace-dummy-%d"
+  organization = tfe_organization.foobar.id
+}
+
+data "tfe_workspace_ids" "good" {
+  tag_names    = ["good"]
+	exclude_tags    = ["happy"]
```

Comment:
Tested with a TFE instance that does **not** support `exclude_tags` to check any other elements would break due to the change.

1.  With `exclude_tags`: argument not supported as expected
_CLI_
<img width="450" alt="Screen Shot 2022-07-08 at 1 45 37 PM" src="https://user-images.githubusercontent.com/59571224/178047206-8dfb95fa-0a9a-4bab-be0e-92a0263be708.png">

_VCS_
<img width="1198" alt="Screen Shot 2022-07-08 at 2 04 37 PM" src="https://user-images.githubusercontent.com/59571224/178047248-fb9c5caa-03b1-4eac-a920-86db84a7a02b.png">

3. Without `exclude_tags`: tfe_workspace resources were still successfully created without error
<img width="1083" alt="Screen Shot 2022-07-08 at 1 46 15 PM" src="https://user-images.githubusercontent.com/59571224/178047326-9f8c405c-9fb8-4566-ae86-434472461205.png">